db/migrate/20161024182548_create_questions.rb

```diff
@@ -0,0 +1,25 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    create_table :questions do |t|
+      t.text            :text
+      t.timestamps null: false
+    end
+
+    create_table :response_choices do |t|
+      t.belongs_to      :questions, index: true
```

Comment:
but don't edit the existing migration file, just make a new migration and rebuild the schema as per @dinahshi 's suggestion below 💃